source/wrapper/reports/SECURITY.md

```diff
@@ -105,6 +114,8 @@ Wrapper.swap(Types.Order) (Full.sol#456-506) ignores return value by external ca
 Reference: https://github.com/crytic/slither/wiki/Detector-Documentation#unused-return
 ```
 
+The WETH contract that is used is known contract with its behavior well understood. The WETH contract will either return true or revert if there is a failure. As far as the code describes, there is no way that false could be returned or that true would be returned without the specified amount of WETH tokens transferred.
```

Comment:
Back on the original PR adding this report to the repo, Don and I had the following convo:
Alice:
> Happy with these invariants. Having reviewed I think they cover everything. Just a couple of thoughts of things we should potentially mention:
> - We didn't write the WETH contract ourselves. However it's publicly verified on etherscan, and is not upgradeable and so we trust it

Don:
> Wrapper does not use a specific WETH contract, only an IWETH interface we developed. Will add a note to suggest that users use the canonical WETH contracts on rinkeby and mainnet.

From this conversation, I'm therefore not sure this addition is necessarily correct - as we don't enforce a specific WETH contract in wrapper? What are your thoughts?

Reply:
If that's the case then we should put a require on the transfer and transferFrom then in the wrapper contract as that statement implies we're treating WETH like a black box and then could have unexpected behavior.

Reply:
In my opinion, we hardcode the weth address so this would be an unnecessary addition. I view it as WETH not IWETH. But let's get @dmosites thoughts on this.

Reply:
i would like to use WETH9.sol then if that's the path forward.

Reply:
Mentioned in the report we should consider adding `require` statements. While we do not use a specific WETH contract, though, I've come to think that it's worth mentioning the WETH contract that we _do_ use for testing and later for deployments, as that's a part of our security posture. For the above addition, I might qualify it as, "The WETH contract used for testing and deployments by the AirSwap team is a known [...]

Reply:
ok, i may have missed that section of the report, I incorrectly thought "If possible, the team should consider ensuring expected behavior with `require` statements." came from Slither.

Reply:
just to close this, i'll remove my accessory comment as it sounds like this has been thought about by the team and just a note in the Notes section is sufficient.